Documentation/applications/examples/elf/index.rst

```diff
@@ -35,28 +42,26 @@ options:
 
    If you use GCC to link, you make also need to include ``-nostdlib``.
 
-3. This example also requires ``genromfs``. ``genromfs`` can be build as part of the
-   nuttx toolchain. Or can built from the ``genromfs`` sources that can be found
-   in the NuttX tools repository (``genromfs-0.5.2.tar.gz``). In any event, the
-   ``PATH`` variable must include the path to the genromfs executable.
-
-4. ELF size: The ELF files in this example are, be default, quite large because
+3. ELF size: The ELF files in this example are, be default, quite large because
```

Comment:
@Laczen maybe for romfs you should keep the information about the need for genromfs

Reply:
Isn't this some old leftover information ? Also for cromfs a tool is needed...